tools/api-markdown-documenter/src/utilities/ApiItemUtilities.ts

```diff
@@ -239,16 +239,57 @@ export function filterByKind(apiItems: readonly ApiItem[], kinds: ApiItemKind[])
 }
 
 /**
- * Gets the release tag associated with the provided API item, if one exists.
+ * Gets the release tag associated with the provided API item, if the item's documentation contained one.
  *
- * @param apiItem - The API item whose documentation is being queried.
+ * @returns The associated release tag, if it exists. Will return `None` if no tag is present.
  *
- * @returns The associated release tag, if it exists. Otherwise, `undefined`.
+ * @privateRemarks
+ * TODO: No one should really use this. They should use `getEffectiveReleaseTag` instead.
+ * This includes the docs we generate - we shouldn't label an interface member as `@public` if the interface itself is
+ * `@beta`, for example, even if that's that member is directly tagged `@public`.
```

Comment:
Also, handled in the FF repo by our API model linting rules, right?
```suggestion
 * `@beta`, for example, even if that member is directly tagged `@public`.
```

Reply:
Yes, these cases will be caught by our API-Extractor linting pass based on our configuration. But this tool is trying to be a bit more general, and the policies we opt into in FF are not strictly required.